Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -1359,6 +1359,14 @@ def generate_enabled_by_for_receiver(receiver, messages, ignore_invalid_message_
     ]
 
 
+def header_for_receiver_name(name):
+    special_headers = {
+        'WebInspector': 'WebInspectorInternal'
+    }
+
+    return special_headers[name] or name
```

Comment:
This is going to raise a `KeyError` if the key `name` is missing from the dictionary.

Reply:
Ah, EWS clean builds show you the problem.